.github/json_matrices/engine-matrix.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "type": "valkey",
+    "version": "7.2.5"
+  },
+  {
+    "type": "valkey",
+    "version": "redis-7.0.15"
+  },
+  {
+    "type": "valkey",
+    "version": "redis-6.2.14"
+  }
+]
```

Comment:
Is it not worth adding 6.0? is there something special about 6.2?

Reply:
Adding 6.0 breaks a lot of our tests due to API incompatibilities. We will do it in a different PR - pre or post GA wasn't decided yet.